Source/g_game.c

```diff
@@ -2722,6 +2740,13 @@ void G_DoNewGame (void)
   deathmatch = false;
   basetic = gametic;             // killough 9/29/98
```

Comment:
I was going to ask why you didn't need to set the basetic in the changes, but I see that it gets set inside G_DoNewGame in woof.

It's in G_DoLoadLevel in pr+ / dsda-doom and with conditions: https://github.com/kraflab/dsda-doom/blob/c482fc2a2e2af8b2a7257dac854fcc507b563d2c/prboom2/src/g_game.c#L1123-L1124

I wonder why this is different :thinking: 

Maybe the way it handles menus and pausing is somehow related :thinking: 

Well, this is probably not an issue with this PR, just a curious observation :snowboarder:

Reply:
Sorry for the late reply, but Woof also have `basetic = gametic;` in `G_DoLoadLevel`: https://github.com/fabiangreffrath/woof/blob/96cc3704893e52fc4b49cc63f5dfee1a7a73347c/Source/g_game.c#L671-L672

Reply:
But this is only for the Boom complevel, right?

Reply:
Yes, it's only for Boom.
The difference is: MBF always set `basetic = gametic` in `G_DoNewGame`, PrBoom+/DSDA-Doom set it only `if (demorecording)` and in `G_DeferedInitNew`.

Reply:
But isn't `G_DoNewGame()` called one tic later than `G_DeferedInitNew()`?

Reply:
Probably, I'm not sure. Is this affect compatibility?

Reply:
> The basetic variable is needed to keep demo sync for Revenant missiles, i.e. it decides whether they are seeking or not.

In Crispy it's called `demostarttic` and is reset in `G_InitNew()`.

Reply:
> In Crispy it's called `demostarttic` and is reset in `G_InitNew()`.

So it's like MBF. So, are we going to keep the MBF code or change it like PRBoom+?

Reply:
I'd say let's keep the MBF code, because this is also responsible for keeping demos intact if the menu is opened during recording. If this breaks demo compatibility should become obvious if you restart a demo on a map with one or more Revenants.